addons/discovery-app/cookiecutter.json

```diff
@@ -0,0 +1,29 @@
+{
+  "project_name": "My Simple App",
+  "project_slug": "@{{ cookiecutter.project_name.lower().replace(' ', '-').replace('_', '-') }}",
+  "github_repo": "@{{ cookiecutter.github_repo }}",
+  "github_org": "@{{ cookiecutter.github_org }}",
+  "app_name": "foo",
+  "aws_region": "eu-central-1",
+  "zone_name": "example.com",
```

Comment:
```suggestion
  "zone_name": "discovery.tamedia.ch",
```
?

Reply:
Actually I can remove it, since not used. The zone_names are already defined in the tfvars files.